opencog/attentionbank/AttentionBank.cc

```diff
@@ -129,6 +129,12 @@ void AttentionBank::stimulate(const Handle& h, double stimulus)
     // XXX This is not protected or made atomic in any way ...
     // If two different threads stimulate the same atom at the same
     // time, then the calculations will be bad. Does it matter?
+    if(stimulusRec.find(h) != stimulusRec.end()){
+        stimulusRec[h] += stimulus;
+    } else{
+        stimulusRec[h] = stimulus;
+    }
+
```

Comment:
Can you put this either above or below the comments?  The comments above your new code refer to the code underneath your addition.